app/manager.py

```diff
@@ -5,7 +5,7 @@
 from config import load_and_validate_rooms
 
 basedir = os.path.abspath(os.path.dirname(__file__))
-configdir = os.path.join(basedir, 'config')
+configdir = os.path.join(basedir, 'config', 'base')
 overridedir = os.path.join(configdir, 'overrides')
```

Comment:
This needs to be updated too now -- probably easiest to just do `os.path.join(basedir, 'config', 'overrides')`?